hota/Mods/hotAcampaignHeroesInRMG/mod.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name" : "Hota campaign Heroes in RMG",
+	"name" : "HotA campaign heroes in RMG",
```

Comment:
It is not "in RMG" but general "in all single player maps".
Random maps, SoD maps
HotA maps are probably unaffected though.